frontend/src/citizen-frontend/personal-details/PersonalDetails.tsx

```diff
@@ -117,10 +117,13 @@ export default React.memo(function PersonalDetails() {
 
             return (
               <>
-                {email === null && (
+                {(!email || !phone) && (
                   <AlertBox
-                    message={t.personalDetails.noEmailAlert}
-                    data-qa="missing-email-box"
+                    message={[
+                      t.personalDetails.noEmailAlert,
+                      t.personalDetails.noPhoneAlert
+                    ].join('\n')}
```

Comment:
This does not actually add a line break in the text. You need a `<br>` element or wrap the sentences in their own `<p>` elements.

Reply:
The alert element (and the stack of component it uses) does not support this, so text formatting is used instead